lib/config.ts

```diff
@@ -14,7 +19,22 @@ if (!GRAPHQL_HOSTNAME) {
   }
 }
 
+if (!GRAPHQL_HOSTNAME_INTERNAL) {
+  if (typeof window !== "undefined") {
+    let hostParts = window.location.host.split(".")
+    if (hostParts.length <= 3) {
+      // throw new Error("Missing env variables")
+      hostParts = "pay.mainnet.galoy.io".split(".")
```

Comment:
Shouldn't this also be changed?

Reply:
to what?

Reply:
Internal dns. Otherwise rate limiting will be applied to `ln.bitcoinbeach.com`.